pkg/eval/builtin_fn_pred.go

```diff
@@ -62,7 +62,9 @@ func init() {
 // ▶ $true
 // ```
 //
-// **NOTE**: `and` and `or` are implemented as special commands.
+// **Note**: `not` is a regular command, and thus can be overriden by a
+// function definition (though you shouldn't do so), while `and` and `or` are
+// implemented as [special commands](language.html#special-commands).
```

Comment:
> and thus can be overriden by a function definition

While this is true and an interesting piece of information, the rationale to mention that `not` is a normal command while `and` and `or` are special commands is certainly **not** to remind the reader that they could technically override `not`. Rather, the rationale is to remind the reader that `and` and `or` actually have special evaluation rules to support short-circuiting, while `not` does not need special evaluation rules.